controllers/elections.py

```diff
@@ -0,0 +1,106 @@
+"""
+Elections page controller.
+"""
+
+__author__ = 'Roshni Kaushik <rsk8@rice.edu>'
+
+import json
+import logging
+import pages
+import webapp2
+from  mail import sendConfirmation
+
+from authentication import auth
+from models.user import User
+
+import models.election
+import models.position
+
+PAGE_URI = '/elections'
+#MY_PAGE_URI = '/my'
+
+class ElectionsHandler(webapp2.RequestHandler):
+		def get(self):
```

Comment:
are these 16 space indents? try to use PEP8 standards (4 space indents)

also there are other PEP8 standards you seem to be missing, if you use pycharm, youll automatically see what guidelines you're breaking. i can get you the professional download code for free if you want